datafusion/core/src/physical_plan/display.rs

```diff
@@ -21,7 +21,7 @@
 
 use std::fmt;
 
-use crate::logical_expr::{StringifiedPlan, ToStringifiedPlan};
+use datafusion_common::display::{StringifiedPlan, ToStringifiedPlan};
```

Comment:
the whole point of this PR is to remove the `logical_expr` from the `use` statements in `physical_plan`